cmd/geoipupdate/main.go

```diff
@@ -17,11 +17,10 @@ var (
 	version                  = "unknown"
 	defaultConfigFile        string
 	defaultDatabaseDirectory string
+	log                      = vars.NewDiscardLogger("main")
 )
 
 func main() {
-	log.SetFlags(0)
```

Comment:
I think the default output format will change with the flags you have in NewDiscardLogger. I don't think we want that.